src/services/ContentService.js

```diff
@@ -1414,8 +1414,25 @@ define(["structures/ContentCreateStruct", "structures/ContentUpdateStruct", "str
      * @method emptyThrash
      * @param callback {Function} callback executed after performing the request (see
      *  {{#crossLink "ContentService"}}Note on the callbacks usage{{/crossLink}} for more info)
+     * @deprecated since 1.1 and will be removed in 2.0. Is replaced `emptyTrash` which has the same behavior
+     *
      */
     ContentService.prototype.emptyThrash = function (callback) {
+        console.warn('[DEPRECATED] ContentService.emptyThrash is deprecated');
+        console.warn('[DEPRECATED] ContentService.emptyThrash is deprecated will be removed in eZ JS REST client 2.0');
```

Comment:
well, actually, we haven't release the REST client so I'm unsure about that. I guess we should tag the 1.0 before this PR and start to use the tags in PlatformUIAssetsBundle instead of using master ?
ping @andrerom @bdunogier